tests/binary.rs

```diff
@@ -74,65 +72,39 @@ mod binary {
 
   #[test]
   fn two_pass_bitrate_based() {
-    let outfile = get_tempfile_path("ivf");
-    let passfile = get_tempfile_path("pass");
+    let outfile = &*get_tempfile_path("ivf");
+    let passfile = &*get_tempfile_path("pass");
 
-    let mut cmd1 = get_rav1e_command();
-    cmd1
-      .arg("--bitrate")
-      .arg("1000")
-      .arg("-o")
-      .arg(&outfile)
-      .arg("--first-pass")
-      .arg(&passfile)
+    get_common_cmd(outfile)
+      .args(&[OsStr::new("--first-pass"), passfile])
       .arg("-")
       .write_stdin(get_y4m_input())
       .assert()
       .success();
 
-    let mut cmd2 = get_rav1e_command();
-    cmd2
-      .arg("--bitrate")
-      .arg("1000")
-      .arg("-o")
-      .arg(&outfile)
-      .arg("--second-pass")
-      .arg(&passfile)
+    get_common_cmd(outfile)
+      .args(&[OsStr::new("--second-pass"), passfile])
       .arg("-")
       .write_stdin(get_y4m_input())
       .assert()
       .success();
   }
   #[test]
   fn two_pass_bitrate_based_constrained() {
-    let outfile = get_tempfile_path("ivf");
-    let passfile = get_tempfile_path("pass");
+    let outfile = &*get_tempfile_path("ivf");
+    let passfile = &*get_tempfile_path("pass");
 
-    let mut cmd1 = get_rav1e_command();
-    cmd1
-      .arg("--reservoir-frame-delay")
-      .arg("14")
-      .arg("--bitrate")
-      .arg("1000")
-      .arg("-o")
-      .arg(&outfile)
-      .arg("--first-pass")
-      .arg(&passfile)
+    get_common_cmd(outfile)
+      .args(&["--reservoir-frame-delay", "14"])
+      .args(&[OsStr::new("--first-pass"), passfile])
```

Comment:
It is more readable to use `arg().arg()` instead.

Reply:
That's true.